lib/vyasa/sangh/worktable.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Vyasa.Sangh.Workspace do
+  use GenServer
+  # a bit like a tuplespace per disciple
+  # store curr_binding state
```

Comment:
essence: 
presence module allows us to not mediate thru a db, small diffs are pubsubed here ==> fundamentally a many-to-many r/s 

this space is where it's mediated through, it's a genserver keyvalue thing

"workspace" is a good name

Reply:
@ks0m1c remember to rename file